README.md

```diff
@@ -51,13 +51,14 @@ The versions of various services used in the docker-compose are listed in the en
 For persistent data, docker volume `mongo_data` is created.
```

Comment:
What does persistent data mean in this case? 
It seems like there are a lot of components created by or part of PEP. We should list them and what they do at the beginning.